cpanfile

```diff
@@ -1,10 +1,12 @@
+requires 'perl', '5.014';
 requires 'Moo';
 requires 'Mojo::UserAgent';
-requires 'Math::BigInt', '>= 1.999811';
-requires 'Math::BigFloat', '>= 1.999811';
-requires 'perl', '5.014';
+requires 'Math::BigInt', '>= 1.999814';
+requires 'Math::BigFloat', '>= 1.999814';
 requires 'JSON', '2.97001';
 requires 'JSON::MaybeXS', '>= 1.003008';
+requires 'List::Util', '1.23';
+requires 'Scalar::Util', '1.23';
```

Comment:
The `requires 'perl', '5.014'` above already implies version 1.23 of both List::Util and Scalar::Util (https://metacpan.org/source/JESSE/perl-5.14.0/cpan/List-Util/lib/List/Util.pm#L17) so probably no need for the explicit version here.